### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by vjik**
- Add more tests for `ArrayHelper::setValue`
- Fix phpdoc
- Change test files location more understandable

### CODE REVIEW

**Review by roxblnfk — COMMENTED** [review]
Can you fix [this comment](https://github.com/yiisoft/arrays/blob/72736b7ece25be4e79b727923e3e91e804130c9a/src/ArrayHelper.php#L106) too?

**Comment by vjik** [comment]
> Can you fix [this comment](https://github.com/yiisoft/arrays/blob/72736b7ece25be4e79b727923e3e91e804130c9a/src/ArrayHelper.php#L106) too?

Done

**Review by xepozz — APPROVED** [review]

File: tests/ArrayHelper/ArrayHelperTest.php [hunk 1]
```diff
@@ -2,13 +2,17 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Arrays\Tests;
+namespace Yiisoft\Arrays\Tests\ArrayHelper;
```
Comment (rustamwin) [inline tests/ArrayHelper/ArrayHelperTest.php:5]: Can we move these just `Helper` directory?

Comment (vjik) [reply tests/ArrayHelper/ArrayHelperTest.php:5]: You mean rename `Yiisoft\Arrays\Tests\ArrayHelper` to `Yiisoft\Arrays\Tests\Helper` ?

Comment (rustamwin) [reply tests/ArrayHelper/ArrayHelperTest.php:5]: Yes.

Comment (vjik) [reply tests/ArrayHelper/ArrayHelperTest.php:5]: I think class name `ArrayHelper` and folder name with his tests should have equal names.

**Review by rustamwin — APPROVED** [review]

**Review by samdark — APPROVED** [review]
Please resolve the conflict. After that, this PR is good to merge. Thanks.

**Comment by vjik** [comment]
> Please resolve the conflict. After that, this PR is good to merge. Thanks.

Done.

**Comment by samdark** [comment]
👍

**Pull request merged**